hikey.mk

```diff
@@ -209,7 +209,6 @@ xtest: xtest-common
 # "make clean" in xtest: fails if optee_os has been cleaned previously
 .PHONY: xtest-clean
 xtest-clean: xtest-clean-common
-	rm -rf $(OPTEE_TEST_OUT_PATH)
```

Comment:
Thanks! There's no harm changing it to that but it's essentially a no ops since it seems like the dir will not be empty. xtest-clean-common removes all the files but leaves all the subdirs intact. Plus all the other platforms don't have an extra command for xtest-clean, so might as well try to keep things inline with the others?